packages/app/src/App.tsx

```diff
@@ -66,6 +69,17 @@ export const AppBase = () => {
               {searchPage}
             </Route>
             <Route path="/catalog-graph" element={<CatalogGraphPage />} />
+            <Route
+              path="/rbac"
+              element={
+                <RequirePermission
+                  permission={policyEntityReadPermission}
+                  resourceRef="policy-entity"
+                >
+                  <RbacPage />
+                </RequirePermission>
+              }
+            />
```

Comment:
good to add these in README and we can remove these changes in `**/app/**`

Reply:
@tumido does the above comment make sense or this is an RBAC plugin so the change is fine?

Reply:
@invincibleJai your previous comment makes sense as this change will not be released with RBAC plugin as its in the core App component and the user will anyways need to add this rbac <Route /> manually in their App component.